extensions/wikia/SpecialEditHub/SpecialEditHub.setup.php

```diff
@@ -46,7 +46,5 @@
 
 $wgGroupPermissions['*']['edithub'] = false;
 $wgGroupPermissions['staff']['edithub'] = true;
-$wgGroupPermissions['sysop']['edithub'] = true;
```

Comment:
CSS changes look fine, the other Permissions change looks fine, can you just remind me why the 'edithub' permission would be removed here?